src/test/java/net/sf/jabref/remote/DBConnectorTest.java

```diff
@@ -0,0 +1,58 @@
+package net.sf.jabref.remote;
+
+import java.sql.SQLException;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class DBConnectorTest {
+
+    @Test
+    public void testGetNewMySQLConnection() {
+        try {
+            DBConnector.getNewConnection(DBType.MYSQL, "localhost", "jabref", "root", "");
+        } catch (Exception e) {
```

Comment:
Here we have the opposite of the regular case. This test is positive only if no exception occured. It doesen't matter which one.